
/home/obatzer/AnalyzerForIDS/node_modules/mongodb/lib/mongo_client.js:236
          throw err
          ^
MongoError: Invalid Operation, No operations in bulk
    at Function.MongoError.create (/home/obatzer/AnalyzerForIDS/node_modules/mongodb-core/lib/error.js:31:11)
    at toError (/home/obatzer/AnalyzerForIDS/node_modules/mongodb/lib/utils.js:114:22)
    at OrderedBulkOperation.execute (/home/obatzer/AnalyzerForIDS/node_modules/mongodb/lib/bulk/ordered.js:503:11)
    at bulkWrite (/home/obatzer/AnalyzerForIDS/node_modules/mongodb/lib/collection.js:653:8)
    at Collection.insertMany (/home/obatzer/AnalyzerForIDS/node_modules/mongodb/lib/collection.js:523:44)
    at Collection.insert (/home/obatzer/AnalyzerForIDS/node_modules/mongodb/lib/collection.js:825:15)
    at /home/obatzer/AnalyzerForIDS/analyzer.js:131:18
    at connectCallback (/home/obatzer/AnalyzerForIDS/node_modules/mongodb/lib/mongo_client.js:314:5)
    at /home/obatzer/AnalyzerForIDS/node_modules/mongodb/lib/mongo_client.js:233:11
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
